extensions/ql-vscode/src/vscode-tests/cli-integration/queries.test.ts

```diff
@@ -53,18 +52,14 @@ describe('Queries', function() {
         qs = extension.qs;
         cli.quiet = true;
         ctx = extension.ctx;
-        qlpackFile = `${ctx.storageUri?.fsPath}/quick-queries/qlpack.yml`;
+        qlpackFile = `${ctx.storageUri?.fsPath}/quick-queries/qlpack-${faker.name}.yml`;
         qlpackLockFile = `${ctx.storageUri?.fsPath}/quick-queries/codeql-pack.lock.yml`;
         oldQlpackLockFile = `${ctx.storageUri?.fsPath}/quick-queries/qlpack.lock.yml`;
-        qlFile = `${ctx.storageUri?.fsPath}/quick-queries/quick-query.ql`;
+        qlFile = `${ctx.storageUri?.fsPath}/quick-queries/quick-query-${faker.name}.ql`;
```

Comment:
Unfortunately, this won't work since we expect a hard coded name for the quick query file.

I think we need to update the `ctx.storageUri` for each test and keep the rest of the path the same.

Reply:
Thanks for addressing this @aeisenberg. Closing this for now. If there are more flakey tests that pop-up I'll have another look. 👀